src/components/Breadcrumb/Breadcrumb.spec.js

```diff
@@ -8,100 +8,100 @@ describe('Breadcrumb Suite', () => {
     const staticText = '[data-qa-label-title]';
     const editableText = '[data-qa-editable-text]';
     const editableTextInput = `${editableText} input`;
-    const editButton = '[data-qa-edit-button]';
+    const editButton = '[data-testid="editable-text"]';
     const saveEditButton = '[data-qa-save-edit]';
     const clearEditButton = '[data-qa-cancel-edit]';
 
-    // it('There should be a link in each Breadcrumb story', () => {
-    //     executeInAllStories(component, childStories, () => {
-    //         $(link).waitForVisible(constants.wait.normal);
-    //         expect($(link).getAttribute('href')).not.toBeNull();
-    //     });
-    // });
+    it('There should be a link in each Breadcrumb story', () => {
+        executeInAllStories(component, childStories, () => {
+            $(link).waitForDisplayed(constants.wait.normal);
+            expect($(link).getAttribute('href')).not.toBeNull();
+        });
+    });
 
-    // it('Static text is not editable, and does not contain link', () => {
-    //     navigateToStory(component, childStories[0]);
-    //     expect($(editButton).isVisible()).toBe(false);
-    //     expect($(staticText).$('..').$('..').getAttribute('href')).toBeNull();
-    // });
+    it('Static text is not editable, and does not contain link', () => {
+        navigateToStory(component, childStories[0]);
+        expect($(editButton).isDisplayed()).toBe(false);
+        expect($(staticText).$('..').$('..').getAttribute('href')).toBeNull();
+    });
 
-    // it('Static text is not editable, and does contain link', () => {
-    //     navigateToStory(component, childStories[1]);
-    //     expect($(editButton).isVisible()).toBe(false);
-    //     expect($(staticText).$('..').$('..').getAttribute('href')).not.toBeNull();
-    // });
+    fit('Static text is not editable, and does contain link', () => {
+        navigateToStory(component, childStories[1]);
+        expect($(editButton).isDisplayed()).toBe(false);
+        expect($(staticText).$('..').$('..').getAttribute('href')).not.toBeNull();
+    });
 
-    // it('Editable text header should be editable, but not contain a link', () => {
-    //     navigateToStory(component, childStories[2]);
-    //     expect($(editButton)).not.toBeNull();
-    //     expect($(editableText).$('..').getAttribute('href')).toBeNull();
-    // });
+    it('Editable text header should be editable, but not contain a link', () => {
+        navigateToStory(component, childStories[2]);
+        expect($(editButton)).not.toBeNull();
```

Comment:
do we want to make any assertions on visibility of this button, or is that flakey because of the hover?

Reply:
The breadcrumb spec has not been updated yet. This is still a work in progress.